echosrv/echo.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	port      = flag.Int("port", 8080, "default http port")
+	port      = flag.String("port", "0.0.0.0:8080", "default http port")
```

Comment:
I would leave the default as `8080` and document in the string that it can be `address:port`